lib/test_queue/iterator.rb

```diff
@@ -64,6 +65,7 @@ def each
       end
     rescue Errno::ENOENT, Errno::ECONNRESET, Errno::ECONNREFUSED
     ensure
+      $0 = procline
```

Comment:
It probably won't stick around, though subclasses could do whatever work they want in `cleanup_worker`.